cmd/api-response.go

```diff
@@ -483,9 +486,12 @@ func generateListVersionsResponse(bucket, prefix, marker, versionIDMarker, delim
 	data.IsTruncated = resp.IsTruncated
 
 	prefixes := make([]CommonPrefix, 0, len(resp.Prefixes))
-	for _, prefix := range resp.Prefixes {
+	for _, cprefix := range resp.Prefixes {
 		var prefixItem = CommonPrefix{}
-		prefixItem.Prefix = s3EncodeName(prefix, encodingType)
+		prefixItem.Prefix = s3EncodeName(cprefix, encodingType)
+		if cprefix == slashSeparator {
```

Comment:
I think you mean prefix here not cprefix

Reply:
it is `cprefix` @vadmeste since that is the `for loop`